pkg/gcptarget/gcptarget_test.go

```diff
@@ -73,7 +73,27 @@ func (td *reviewTestData) assetTestcase() *targetHandlerTest.ReviewTestcase {
 	return tc
 }
 
-var testData = []reviewTestData{
+func (td *reviewTestData) legacySpecMatchTestcase() *targetHandlerTest.ReviewTestcase {
+	legacyMatch := map[string]interface{}{}
+
+	if ancestries, ok := td.match["ancestries"]; ok {
+		legacyMatch["target"] = ancestries
+	}
+	if excludedAncestries, ok := td.match["excludedAncestries"]; ok {
+		legacyMatch["exclude"] = excludedAncestries
+	}
+
+	tc := &targetHandlerTest.ReviewTestcase{
+		Name:                "legacy spec match " + td.name,
+		Match:               legacyMatch,
+		WantMatch:           td.wantMatch,
+		WantConstraintError: td.wantConstraintError,
+	}
+	tc.Object = asset(td.ancestryPath)
+	return tc
+}
+
+var matchTests = []reviewTestData{
```

Comment:
Is there a couple of test cases to ensure we're getting a warning if using the legacy `target`?

Reply:
Added in f3addb9